intelmq/bots/collectors/mail/collector_mail_attach.py

```diff
@@ -88,6 +91,11 @@ def process_message(self, uid, message):
                         else:
                             self.logger.error('Could not decrypt attachment %s: %s.', file_name, gpg.status)
                             continue
+
+                    if not raw_report and self.allow_empty:
+                        self.logger.info("Email report read (empty).")
```

Comment:
Also log the action (ignored):
```suggestion
                        self.logger.info("Empty email report ignored.")
```